models/articles.models.js

```diff
@@ -1,5 +1,18 @@
 const db = require("../db/connection");
 
+//SELECT
+
+exports.selectArticles = async () => {
+  const result = await db.query(`SELECT articles.*, 
```

Comment:
how can we change this so we don't get the body

Reply:
I need to read the task description more carefully. Removed now.